pkg/publicapi/endpoint/orchestrator/job.go

```diff
@@ -76,8 +76,10 @@ func (e *Endpoint) getJob(c echo.Context) error { //nolint: gocyclo
 	if err := c.Bind(&args); err != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, err.Error())
 	}
+	log.Info().Msg("UDIT SUCKS")
 	job, err := e.store.GetJob(ctx, jobID)
 	if err != nil {
+		log.Error().Err(err)
```

Comment:
do we need to log the error here? might cause a lot of noise

Reply:
Done